flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/test/java/org/apache/flink/stormcompatibility/api/FlinkTestCluster.java

```diff
@@ -46,15 +52,24 @@ public void submitTopologyWithOpts(final String topologyName, final Map<?, ?> co
 			final SubmitOptions submitOpts)
 			throws Exception {
 		final TestStreamEnvironment env = (TestStreamEnvironment) StreamExecutionEnvironment.getExecutionEnvironment();
-		env.start(topology.getStreamGraph().getJobGraph(topologyName));
+
+		JobGraph jobGraph = topology.getStreamGraph().getJobGraph(topologyName);
+		this.jobIdBuffer.put(topologyName, jobGraph.getJobID());
+
+		env.start(jobGraph);
 	}
 
 	@Override
 	public void killTopology(final String topologyName) {
+		this.killTopologyWithOpts(topologyName, null);
 	}
 
 	@Override
 	public void killTopologyWithOpts(final String name, final KillOptions options) {
+		// TODO: consider options
```

Comment:
I feel quite responsible for Storm compatibility layer and have it in mind. Nevertheless, I will extend this comment to make sure :)